go.mod

```diff
@@ -18,15 +18,23 @@ require (
 )
 
 require (
-	github.com/go-task/slim-sprig v0.0.0-20210107165309-348f09dbbbc0 // indirect
-	golang.org/x/tools v0.1.10-0.20220218145154-897bd77cd717 // indirect
+	code.gitea.io/sdk/gitea v0.14.0 // indirect
```

Comment:
Please make sure, that all indirect dependencies are under the same require block